jet/kafka-connect/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-3.9.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.hazelcast</groupId>
+    <artifactId>jet-kafka-connect</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>com.hazelcast.jet</groupId>
+            <artifactId>hazelcast-jet-kafka-connect</artifactId>
+            <version>5.3.0-BETA-1</version>
```

Comment:
This should use `${hazelcast.version}`, makes it easier to update across all samples.

Reply:
Fixed https://github.com/hazelcast/hazelcast-code-samples/pull/569/commits/86ed90186d28df6a0ea21cbedafcd2513964eef5